web-console/src/views/workbench-view/explain-dialog/__snapshots__/explain-dialog.spec.tsx.snap

```diff
@@ -475,7 +475,7 @@ exports[`ExplainDialog matches snapshot on some data (one query) 1`] = `
         label="Signature"
       >
         <Blueprint4.InputGroup
-          defaultValue="d0::STRING, a0::LONG"
+          defaultValue="d0::STRING→channel, a0::LONG→"Count""
```

Comment:
```suggestion
          defaultValue="d0::STRING→channel, a0::LONG→\"Count\""
```

Reply:
That is a snapshot. It is auto generated. There was a merge conflict with master which I just resolved.